subprojects/docs/src/snippets/multiproject/dependencies-outgoingArtifact/groovy/producer/build.gradle

```diff
@@ -1,10 +1,16 @@
+plugins {
+    id 'java-library'
+}
+
+version = '1.0'
+
 task buildInfo(type: BuildInfo) {
-    version = project.version
-    outputFile = file("$buildDir/generated-resources/build-info.properties")
+    version.set(project.version)
+    outputFile.set(file("$buildDir/generated-resources/build-info.properties"))
```

Comment:
I _think_ for consistency we should keep using the `=` notation on Groovy DSL. Because we use it in all other places (samples, init plugin, ...) as well.